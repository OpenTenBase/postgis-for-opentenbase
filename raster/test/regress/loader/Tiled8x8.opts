-t 8x8
