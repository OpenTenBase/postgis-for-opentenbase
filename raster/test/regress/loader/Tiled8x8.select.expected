0|||||f|f|||||
POLYGON((88 0,89 0,89 -1,88 -1,88 0))|0
POLYGON((88 0,89 0,89 -1,88 -1,88 0))|90
POLYGON((56 -24,57 -24,57 -25,56 -25,56 -24))|255
POLYGON((88 -48,89 -48,89 -49,88 -49,88 -48))|198
